pkg/apis/testmachinery/v1beta1/types.go

```diff
@@ -135,6 +135,9 @@ type TestrunStatus struct {
 	// This value is calculated by (CompletionTime - StartTime)
 	Duration int64 `json:"duration,omitempty"`
 
+	// Tells whether testrun failed first and succeeded in a retry
+	Flaked bool `json:"flaked,omitempty"`
```

Comment:
do not make it part of the testrun resource as this parameter is only known by the testrunner

maybe rather add it to the Run or directly to the metadata

Reply:
done